pkg/controller/migmigration/backup.go

```diff
@@ -114,6 +137,14 @@ func (t *Task) ensureStageBackup() (*velero.Backup, error) {
 	newBackup.Spec.IncludedResources = toStringSlice(includedResources.Difference(toSet(t.PlanResources.MigPlan.Status.ExcludedResources)))
 	newBackup.Spec.ExcludedResources = toStringSlice(settings.ExcludedStageResources.Union(toSet(t.PlanResources.MigPlan.Status.ExcludedResources)))
 	newBackup.Spec.LabelSelector = &labelSelector
+
```

Comment:
A few lines up, there is code which excludes ImageStreams from stage backup if we're doing direct image migration. We actually need to *include* these in the backup if DIM is configured *and* DISABLE_IMAGE_COPY is set (otherwise if the DIM flag is set on the MigPlan, we won't create the placeholder. Basically the below:
```
	if t.indirectImageMigration() {
		includedResources = settings.IncludedStageResources
	} else {
		includedResources = settings.IncludedStageResources.Difference(mapset.NewSetFromSlice([]interface{}{settings.ISResource}))
	}
￼```
Should probably be replaced with something like this:
```
	if t.indirectImageMigration() || imageCopyFlag {
		includedResources = settings.IncludedStageResources
	} else {
		includedResources = settings.IncludedStageResources.Difference(mapset.NewSetFromSlice([]interface{}{settings.ISResource}))
```

Reply:
@sseago This change is done.